alpha-build-machinery/make/targets/golang/verify-update.mk

```diff
@@ -26,6 +26,16 @@ verify-golint:
 	$(GOLINT) $(GO_PACKAGES)
 .PHONY: verify-govet
 
+verify-sec:
+ifeq (, $(shell which $(GOSEC) 2>/dev/null))
+	GO111MODULE=on go get github.com/securego/gosec/cmd/gosec@4b59c948083cd711b6a8aac8f32721b164899f57
+endif
+	$(GOSEC) \
+		-severity $(GOSEC_SEVERITY) -confidence $(GOSEC_CONFIDENCE) \
```

Comment:
no default value?

Reply:
the default value is in another file (similar to the default value of GOFMT_FLAGS)

Reply:
They are set in aonther file, similar to GOFMT_FLAGS